### ISSUE TITLE
Dynamically expand/contract accordion

### ISSUE THREAD

**Issue by cusebassman** [uikit/uikit#995]
Hello all - I'm curious if anyone knows how I might go about calling a JavaScript function to automatically resize the visible section of a UIKit accordion instance when I use JavaScript to add content to the visible section.
Right now I have a page that contains an accordion, and inside each uk-accordion-content div I have an unordered list that a button can add list items to. When the JavaScript adds a list item, the visible accordion section expands beyond(and underneath) the next, closed accordion section.
I've tried reviewing Firebug stack traces to figure out the function calls I might need, but I'm a bit lost. Thanks for any advice you might have!

**Comment by malles** [comment]
You'll be looking for http://getuikit.com/docs/documentation_javascript.html#dom-observe. Or found it, since this issue is already closed.

**Comment by cusebassman** [comment]
Malles, thanks for replying anyway. I closed the issue originally because I thought I'd figured out how to modify the accordion jS to do what I needed. When I saw your reply, I tried the dom observe method, but that doesn't appear to work.
It works for initializing UIKIT objects that I add dynamically to the page through JavaScript, but it doesn't seem to work as far as updating the vertical size of the visible accordion content when I use JavaScript to add content to the visible accordion pane.
I'll see if I can post the page I'm having issues with later this evening.
